ra2ce/analysis/damages/damages.py

```diff
@@ -46,6 +52,10 @@ def __init__(
         self.input_path = analysis_input.input_path
         self.output_path = analysis_input.output_path
         self.reference_base_graph_hazard = base_graph_hazard
+        if self.analysis.damage_curve == DamageCurveEnum.MAN:
+            self.manual_damage_functions = ManualDamageFunctionsReader().read(
+                self.input_path.joinpath("damage_functions")
+            )
```

Comment:
I might be wrong, but I believe this was placed on purpose under `execute` as you only need it then, thus reducing the memory usage when initializing this object. (Do not do computations / use memory if not needed).

Reply:
*Note, perhaps good to mention that a comment with my previous explanation might be useful to avoid the same mistake again.

Reply:
The goal of this PR is to read it as part of the initialization, so this is done on purpose.